doc/sphinx-guides/source/user/dataset-management.rst

```diff
@@ -683,17 +683,26 @@ If you have a Contributor role (can edit metadata, upload files, and edit files,
 Preview URL to Review Unpublished Dataset
 =========================================
 
-Creating a Preview URL for your dataset allows you to share your dataset (for viewing and downloading of files) before it is published to a wide group of individuals who may not have a user account on the Dataverse installation. Anyone you send the Preview URL to will not have to log into the Dataverse installation to view the dataset.
+Creating a Preview URL for a draft version your dataset allows you to share your dataset (for viewing and downloading of files) before it is published to a wide group of individuals who may not have a user account on the Dataverse installation. Anyone you send the Preview URL to will not have to log into the Dataverse installation to view the unpublished dataset. Once a dataset has been published you may continue to use the General Preview URL for future drafts, but the Anonymous Preview URL will no longer be available.
```

Comment:
Interesting. I seem to remember that Preview URLs were deleted on publish but apparently that isn't the case anymore, if it ever was!

Reply:
Maybe unclear. Trying to say that you can make a new ":general" preview url for subsequent drafts. Didn't mean to say that the prior one would continue to work. I'll re-word it.